src/main/java/Task.java

```diff
@@ -0,0 +1,41 @@
+public class Task {
+    private String description;
+    private boolean isDone;
+    private int taskNumber;
+    private static int numberOfTasks = 0;
+
+    public Task(String description){
+        numberOfTasks += 1;
+        this.description = description;
+        this.isDone = false;
+        this.taskNumber = numberOfTasks;
+    }
+    public void markDone(){
+        isDone = true;
+    }
+
+    public void markUndone() {
```

Comment:
Perhaps you would like to have a setter function instead to reduce repeated code?

```suggestion
    public void setDone(boolean value) {
```